core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -119,10 +125,9 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     }
     val allowMatch = allowOps.exists(operation => aclMatch(operation, resource, principal, host, Allow, acls))
 
-    //we allow an operation if a user is a super user or if no acls are found and user has configured to allow all users
+    //we allow an operation if no acls are found and user has configured to allow all users
     //when no acls are found or if no deny acls are found and at least one allow acls matches.
-    val authorized = isSuperUser(operation, resource, principal, host) ||
-      isEmptyAclAndAuthorized(operation, resource, principal, host, acls) ||
+    val authorized = isEmptyAclAndAuthorized(operation, resource, principal, host, acls) ||
       (!denyMatch && allowMatch)
 
     logAuditMessage(principal, authorized, operation, resource, host)
```

Comment:
looks like we miss audit messages for super users. we need to call logAuditMessage in above check.